pkg/apply/apply.go

```diff
@@ -437,16 +470,24 @@ func (t *TopicApplier) updateSettings(
 
 	if len(configEntries) > 0 {
 		if t.config.DryRun {
+			if len(missingKeys) > 0 {
+				log.Warnf(
+					"Found %d key(s) set in cluster but missing from config:\n%s\nThese will be left as-is.",
+					len(missingKeys),
+					FormatMissingKeys(topicInfo.Config, missingKeys),
+				)
```

Comment:
Why was this added here ?

Reply:
currently if we're doing a dry run we don't notify the user if there are `missingKeys` (we return before we hit https://github.com/getsentry/topicctl/pull/3/files#diff-e93827340ebe6b239ff60427dba9f2881d96e7bba9a8a9b07a09cb64582dd73fR507).

I'll move things around so it's more elegant